pkg/watermarker/watermarker.go

```diff
@@ -0,0 +1,30 @@
+package watermarking
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	version "github.com/openshift/must-gather-clean/pkg/version"
+)
+
+type WaterMarker interface {
+	// WriteWaterMarkFile creates a watermark file in the specified path
+	WriteWaterMarkFile(path string) error
+}
+
+type SimpleWaterMarker struct{}
+
+func NewSimpleWaterMarker() *SimpleWaterMarker {
+	return &SimpleWaterMarker{}
+}
+
+func (s *SimpleWaterMarker) WriteWaterMarkFile(path string) error {
+	contents := time.Now().UTC().String() + " - " + version.GetVersion().Version
```

Comment:
super small nitpick: can we add those onto separate lines? just in case somebody wants to read this automatically

Reply:
Sure. done

Reply:
@sherine-k , sorry I wasn't clear. I meant having the content separated by newlines, not the variables :D